core/controller/src/main/scala/whisk/core/controller/actions/PrimitiveActions.scala

```diff
@@ -613,7 +613,13 @@ protected[actions] trait PrimitiveActions {
     if (!result.isCompleted && retries < maxRetries) {
       val schedule = actorSystem.scheduler.scheduleOnce(wait(retries)) {
         activationStore.get(ActivationId(docid.asString), context).onComplete {
-          case Success(activation)             => result.trySuccess(Right(activation))
+          case Success(activation) =>
```

Comment:
do you just want to count the number of times the result is retrieved from the DB directly vs active ack, or the total number of times this poller is sheduled?

Reply:
@rabbah, I just want to count how many times the polling mechanism is actually used to get the activation instead of the active ack.